<launch>

    <node name="lidar" pkg="lidar" type="lidar" output="screen">
        <param name="scan_topic" value="scan"/>             # 设置激光数据topic名称
        <param name="laser_link" value="laser"/>            # 激光坐标
        <param name="serial_port" value="/dev/laser"/>
        <param name="zero_as_max" value="false"/>           # 设置为true探测不到区域会变成最大值
        <param name="min_as_zero" value="false"/>           # true：探测不到区域为0，false：探测不到区域为inf
        <param name="angle_disable_min" value="-1"/>        # 角度制，从angle_disable_min到angle_disable_max之前的值为0
        <param name="angle_disable_max" value="-1"/>
        <param name="inverted" value="false"/>              # 如果0度方向在串口线的方向上设置为true 
    </node>
</launch>